app/src/components/ViewEventsCards/ViewEventsCardsContainer.tsx

```diff
@@ -3,28 +3,40 @@ import style from './ViewEventsCards.module.scss';
 import { createRoute } from 'src/routing';
 import { hasLoaded } from 'src/remote-data';
 import { Page } from 'src/components/Page/Page';
-import { useEvents } from 'src/hooks/cache';
+import { useEvents, usePastEvents } from 'src/hooks/cache';
 import { EventCardElement } from 'src/components/ViewEventsCards/EventCardElement';
 import { Button } from 'src/components/Common/Button/Button';
 import { useHistory } from 'react-router';
 import { authenticateUser, isAuthenticated } from 'src/auth';
 
 export const ViewEventsCardsContainer = () => {
   const events = useEvents();
+  const pastEvents = usePastEvents();
```

Comment:
Nå ender vi jo opp med to nettverkskall. Du nevnte at dette er futureproofing, at vi mest sannsynlig trenger det i fremtiden med pagination etc. Men jeg tenker muligens query parameters kan brukes til det.

I dette tilfellet kan frontend skille mellom kommende og tidligere arrangmenter.